src/classes/BDI_DataImportService_TEST.cls

```diff
@@ -113,6 +113,116 @@ private with sharing class BDI_DataImportService_TEST {
         }
     }
 
+    /*******************************************************************************************************************
+    * @description Verifies correct working using the custom metadata instead of help text.
+    *
+    */
+    @isTest
+    private static void useCustomMetatadaInsteadOfHelpText() {
+
+        List<DataImport__c> dataImports = new List<DataImport__c>();
+
+        BDISettings__c bdiSettings = BDISettings__c.getOrgDefaults();
+
+        bdiSettings.UseCustomMetadatainsteadOfHelpText__c = true;
+
+        upsert bdiSettings;
+
+        Boolean isDryRun = false;
+
+        BDI_DataImportService service = new BDI_DataImportService(isDryRun);
+
+        DataImport__c firstDataImport = new DataImport__c(
+           Contact1_Firstname__c = 'John',
+           Contact1_Lastname__c = 'Doe',
+           Contact1_Personal_Email__c = 'john@doe.com'
+        );
+
+        DataImport__c secondDataImport = new DataImport__c(
+           Contact1_Firstname__c = 'John',
+           Contact1_Lastname__c = 'Doe',
+           Contact1_Personal_Email__c = 'john@doe.com'
+        );
+
+        dataImports.add(firstDataImport);
+        dataImports.add(secondDataImport);
+
+        insert dataImports;
+
+        System.assertEquals(firstDataImport.status__c,  null);
```

Comment:
status__c -> Status__c

Reply:
There a few more lowercase references remaining in the file.